### DISCUSSION TITLE
用PubLayNet数据集训练ppstructure无法识别表格

### DISCUSSION THREAD

**Discussion by bekars** [PaddlePaddle/PaddleOCR#12691] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu22.04 Nvidia A6000 GPU
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：PaddleDetection release/2.7
- 运行指令/Command Code：

按README文件里的训练过程训练：https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/ppstructure/layout/README_ch.md

配置文件只修改了PubLayNet数据集的目录，其他配置都没有改。

```
export CUDA_VISIBLE_DEVICES=0
python3 tools/train.py \
    -c configs/picodet/legacy_model/application/layout_analysis/picodet_lcnet_x1_0_layout.yml \
    --eval
```

- 完整报错/Complete Error Message：不能识别表格，Table被认为是Figure

<img width="586" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/2306503/d6deb309-60ee-4454-ba9e-483f74cbdf60">

**Comment by tink2123** [comment]
检查下数据标签是正确的吗？训练集中表格标注的类别idx是多少，对应字典中的位置是否是table

**Comment by bekars** [comment]
换了一台机器训练就可以识别了，所有训练参数都一样，都是默认的，只是显卡不同。
